sys/tracefd/include/tracefd.h

```diff
@@ -64,15 +74,15 @@ extern "C" {
  * @brief Interval in ms between two flush of all files
  */
 #ifndef TRACEFD_FLUSH_INTERVAL
-#  define TRACEFD_FLUSH_INTERVAL 1000
+#  define TRACEFD_FLUSH_INTERVAL 50
 #endif
 
 /**
  * @def TRACEFD_THREAD_PRIORITY
  * @brief Files flusher thread priority
  */
 #ifndef TRACEFD_THREAD_PRIORITY
-#  define TRACEFD_THREAD_PRIORITY THREAD_PRIORITY_MIN
+#  define TRACEFD_THREAD_PRIORITY (THREAD_PRIORITY_MAIN - 1)
```

Comment:
why not THREAD_PRIORITY_MIN ?

Reply:
Because otherwise the thread doesn't start...